calico/felix/ipsets.py

```diff
@@ -131,7 +120,8 @@ def cleanup(self):
         felix_ipsets = set([n for n in all_ipsets if n.startswith(pfx) or
                                                      n.startswith(tmppfx)])
         whitelist = set()
-        for ipset in self.objects_by_id.values():
+        for ipset in (self.objects_by_id.values() +
+                      self.stopping_objects_by_id.values()):
```

Comment:
I think this was another potential race between the start-of-day cleanup code and the stopping ipset cleanup code.  (I believe it would be benign since one of the cleanups would win and we won't start another ipset until both cleanups finish but better safe than sorry.)